Makefile

```diff
@@ -19,3 +27,32 @@ lint_terraform:
 
 lint_shellcheck:
 	find . -name '*.sh' -print0 | xargs -0 $(SHELLCHECK)
+
+dev: check-env-vars set_aws_account_dev deploy_cloudfoundry_pipeline ## Deploy Pipelines to Dev Environment
+
+ci: check-env-vars set_aws_account_ci set_auto_trigger disable_auto_delete deploy_cloudfoundry_pipeline  ## Deploy Pipelines to CI Environment
+
+stage: check-env-vars set_aws_account_stage disable_auto_delete set_auto_trigger deploy_cloudfoundry_pipeline  ## Deploy Pipelines to Staging Environment
+
+prod: check-env-vars set_aws_account_prod disable_auto_delete set_auto_trigger deploy_cloudfoundry_pipeline  ## Deploy Pipelines to Production Environment
+
+set_aws_account_dev:
+	$(eval export AWS_ACCOUNT=dev)
+
+set_aws_account_ci:
+	$(eval export AWS_ACCOUNT=ci)
+
+set_aws_account_stage:
+	$(eval export AWS_ACCOUNT=stage)
+
+set_aws_account_prod:
+	$(eval export AWS_ACCOUNT=prod)
+
+set_auto_trigger:
+	$(eval export ENABLE_AUTO_DEPLOY=true)
+
+disable_auto_delete:
+	$(eval export DISABLE_AUTODELETE=1)
+
+deploy_cloudfoundry_pipeline:
+	concourse/scripts/pipelines-cloudfoundry.sh
```

Comment:
I think this should deploy all the pipelines (bosh and CF), and therefore be named `deploy_pipelines`.